packages/@aws-cdk/aws-fsx/lib/file-system.ts

```diff
@@ -0,0 +1,87 @@
+import {Connections, IConnectable, ISecurityGroup,  IVpc} from "@aws-cdk/aws-ec2";
+import {IKey} from "@aws-cdk/aws-kms";
+import {Resource} from "@aws-cdk/core";
+
+/**
+ * Interface to implement FSx File Systems.
+ */
+export interface IFsxFileSystem extends IConnectable {
```

Comment:
Why are all classes called `FsxXxx`?

It seems the underlying CloudFormation resources aren't, and one would think it's obvious enough from the fact that they live in the `aws-fsx` library.

Reply:
I've removed the Fsx from all the interface/class/properties names.